src/ProjectNeon/Assets/Data/Templates/Team.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Team : Target
+{
+    public Team(List<Member> members)
```

Comment:
`ctor` doesn't work in the editor.

If we want to be able to initialize this from code change this to an `Init` method.

Reply:
@SilasReinagel Done